docs/SetupAzureDatabase.md

```diff
@@ -51,8 +51,126 @@ Autoscale provisioned throughput in Azure Cosmos DB allows you to scale the thro
+
+### Backup and Restore
+
+Azure Database for MySQL automatically creates server backups and stores them in user configured locally redundant or geo-redundant storage. Backups can be used to restore your server to a point-in-time. 
+
+More information can be found in [Backup and restore in Azure Database for MySQL](https://docs.microsoft.com/en-us/azure/mysql/concepts-backup) documentation.
+
+### Monitor MySQL
+
+By default, monitoring is enabled on Azure Database for MySQL servers.
+
+More information can be found in [Monitoring in Azure Database for MySQL](https://docs.microsoft.com/en-us/azure/mysql/concepts-monitoring) documentation.
+
+## PostgreSQL
+
+In this section, you will set up an Azure Database for PostgreSQL servers for Scalar DL.
+
+### Requirements
+
+* You must create an Azure Database for PostgreSQL servers with the plan `Single server`.
+
+### Steps
+
+* Create an Azure Database for PostgreSQL servers on the base of [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal).
+
+### Configure Scalar DL 
+
+To apply the Scalar DL schema to PostgreSQL, update the following configuration in [schema-loading-custom-values](https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/schema-loading-custom-values.yaml)
+
+```yaml
+database: jdbc
+contactPoints: <PostgreSQL connection url>
+username: <PostgreSQL username>
+password: <PostgreSQL password>
+```
+
+To deploy Scalar DL on PostgreSQL, update the following configuration in [scalardl-custom-values](https://github.com/scalar-labs/scalar-kubernetes/blob/master/conf/scalardl-custom-values.yaml) 
+
+```yaml
+dbContactPoints: <PostgreSQL connection url>
+dbUsername: <PostgreSQL username>
+dbPassword: <PostgreSQL password>
+dbStorage: jdbc
+```
+
+### Scale resources
+
+After you create your server, you can independently change the vCores, the hardware generation, the pricing tier (except to and from Basic), the amount of storage, and the backup retention period. 
+The number of vCores can be scaled up or down. The storage size can only be increased. Scaling of the resources can be done either through the portal or Azure CLI.
+
+#### Storage auto-grow
+
+Storage auto-grow prevents your server from running out of storage and becoming read-only. If storage auto grow is enabled, the storage automatically grows without impacting the workload. 
+For servers with less than equal to 100 GB provisioned storage, the provisioned storage size is increased by 5 GB when the free storage is below 10% of the provisioned storage.
+For servers with more than 100 GB of provisioned storage, the provisioned storage size is increased by 5% when the free storage space is below the greater of 10 GB or 5% of the provisioned storage size. 
+
+### Backup and Restore
+
+Azure Database for PostgreSQL automatically creates server backups and stores them in user configured locally redundant or geo-redundant storage. 
```

Comment:
@brfrn169 Can you check if it is correct?

Reply:
If the Backup in Azure Database for PostgreSQL is a consistent snapshot, it is correct.

`pg_dump` can produce consistent backups according to the following article:
https://dba.stackexchange.com/questions/13742/does-running-pg-dump-on-live-db-produce-consistent-backups/13751

I think we can ask Azure Support if the Backup in Azure Database for PostgreSQL is a consistent snapshot similar to `pg_dump`.

Reply:
Let me check again and update